features/alarm-change-notifier/main-module/src/test/java/org/opennms/plugins/dbnotifier/test/manual/DbChangeNotifierTest.java

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2002-2014 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2014 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.plugins.dbnotifier.test.manual;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import org.opennms.plugins.dbnotifier.DatabaseChangeNotifier;
+import org.opennms.plugins.dbnotifier.DbNotificationClientQueueImpl;
+import org.opennms.plugins.dbnotifier.DbNotifierDataSourceFactory;
+import org.opennms.plugins.dbnotifier.NotificationClient;
+import org.opennms.plugins.dbnotifier.alarmnotifier.AlarmChangeNotificationClient;
+
+
+public class DbChangeNotifierTest {
```

Comment:
This test will never fail since all of the exception are caught. It also assumes that an existing database is provisioned on the localhost, which may not be the case.